assets/javascripts/discourse/connectors/user-card-metadata/gamification-score.hbs

```diff
@@ -1,6 +1,8 @@
 {{#if user.gamification_score}}
   <h3>
     <span class="desc">{{i18n "gamification.score"}}</span>
-    {{fullnumber user.gamification_score}}
```

Comment:
What protections do we have when the global leaderboard does not exist? Can you add a test for this please?

Reply:
I do not have anything for it right now. I will work on it! Thank you!

Reply:
Tests are added!